sycl/doc/extensions/experimental/sycl_ext_oneapi_matrix/sycl_ext_oneapi_matrix.asciidoc

```diff
@@ -110,157 +145,204 @@ enum class layout {
 }
 ```
 
+==== Group Memory Scope
+In this API, we use the terminology of `joint_matrix` instead of plain
+`matrix` to emphasize that the matrix is shared among a group of work
+items and is not private to each work item. The group scope is added
+as an additional template parameter.
 
-#### Group Memory Scope
-In this API, we use the terminology of `joint_matrix` instead of plain `matrix` to emphasize that the matrix is shared among a group of work items and is not private to each work item. The group scope is added as an additional template parameter and is also part of the constructor arguments.
-
-IMPORTANT: In the current implementation, only the `sub_group` scope is supported
+IMPORTANT: In the current implementation, only the `sub_group` scope
+is supported
```

Comment:
No I don't anticipate work group support to be done before that. This is similar to sizes and types that are implementation defined. That's why we provide scopes query in the device descriptors.

Reply:
In this case, I think it would be better to remove these IMPORTANT notes and simply state that the `Group` template parameter must be `sub_group`.  You can state this requirement in the section "Group Memory Scope".

As we discussed in the SPV meeting, it easy to relax requirements in a future version of the spec.  If we decide to support other groups in the future, it's easy to modify the spec to say this.

You raise an interesting point about the query. Presumably, all devices that support joint matrix support the `sub_group` execution scope right now, and they don't support any other execution scope.  Therefore, I think it makes sense to remove that query from this version of the spec.  If we add other execution scopes in future versions of the spec, we can decide at that time whether it makes sense to add a query.  For example, if all devices support the same set of execution scopes, there's no need to ever add a query.  A query is only useful if the set of execution scopes is different from some devices, and it's not clear at this point whether that will be the case.

Reply:
Yes, we should be able to remove these notes and state it as you suggested since the only scope currently supported is subgroup.

@JackAKirk, can you please confirm that's the expectation from the CUDA backend as well? 

I will also remove the scope query.